terraform/startup_script.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+echo "Made it to the user_data 12345"
+echo "$PWD"
+sudo supervisorctl stop all
+sudo supervisorctl status
+mv /home/ubuntu/linc-cv/ /home/ubuntu/linc-cv-backup
+cd /home/ubuntu/
+git clone https://github.com/linc-lion/linc-cv
```

Comment:
Are we going to create another startup script for staging to pull from staging branch?

Reply:
Not sure if there is a way to pas in an argument to execute startup script. A duplicate copy of it is not the end of the world.